app/Services/ModuleService.php

```diff
@@ -240,6 +240,9 @@ public function updateModule($id, $status): bool
         $module->update([
             'enabled' => $status,
         ]);
+        if ($status === 1) {
```

Comment:
check for `true`

Reply:
And add a newline above this one, hard to read without the spacing

Reply:
Alright